package.json

```diff
@@ -69,10 +69,13 @@
     "lint:fix": "standard --fix | snazzy",
     "test": "npm run test:javascript && cross-env NODE_V8_COVERAGE=  npm run test:typescript",
     "test:javascript": "node scripts/generate-pem && npm run test:unit && npm run test:node-fetch && npm run test:fetch && npm run test:cookies && npm run test:eventsource && npm run test:wpt && npm run test:websocket && npm run test:node-test && npm run test:jest",
+    "test:javascript:withoutintl": "node scripts/generate-pem && npm run test:unit && npm run test:node-fetch && npm run test:fetch:nobuild && npm run test:cookies && npm run test:eventsource:nobuild && npm run test:wpt:withoutintl && npm run test:node-test",
     "test:cookies": "borp -p \"test/cookie/*.js\"",
     "test:node-fetch": "borp -p \"test/node-fetch/**/*.js\"",
-    "test:eventsource": "npm run build:node && borp --expose-gc -p \"test/eventsource/*.js\"",
-    "test:fetch": "npm run build:node && borp --expose-gc -p \"test/fetch/*.js\" && borp -p \"test/webidl/*.js\" && borp -p \"test/busboy/*.js\"",
+    "test:eventsource": "npm run build:node && npm run test:eventsource:nobuild",
```

Comment:
why is building bad?

Reply:
Build fails with icu-less node build:

<img width="1274" alt="CleanShot 2024-03-29 at 14 25 05@2x" src="https://github.com/nodejs/undici/assets/1062734/e95bb636-82aa-4f2c-95b3-cd7afa1c8bc4">

But we shouldn't to support _building_ undici with icu-less node, only running it.

This is why the workflow [installs regular node then builds undici](https://github.com/nodejs/undici/pull/3015/files#diff-e98936aa52a6dd7416e4296e9628456227d834f7245967383fd9ff80fd985dadR88) prior to switching to the icu-less build for running tests.

I tried to minimize changes to package.json, so as to make people not need to think about keeping the `test:javascript:withoutintl` script up-to-date with test strategy changes.